src/uniprotkb/components/entry/__tests__/__mocks__/similarProteinsData.json

```diff
@@ -1,267 +0,0 @@
-{
```

Comment:
Wasn't sure if you planned on replacing this rather than deleting? Or will this come later?

Reply:
Good catch, I didn't mean to remove it completely, I'll add it back in TS version

Reply:
back